### PR TITLE
add cxe-red to react-shared-context ownership

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -146,7 +146,7 @@ packages/react-icons-mdl2 @microsoft/cxe-red @microsoft/cxe-coastal
 packages/react-monaco-editor @microsoft/fluentui-v8-website
 packages/react-components/react-positioning @microsoft/teams-prg
 packages/react-components/react-overflow @microsoft/teams-prg
-packages/react-components/react-shared-contexts @microsoft/teams-prg
+packages/react-components/react-shared-contexts @microsoft/teams-prg @microsoft/teams-red @microsoft/teams-coastal
```
Comment (JustSlone) [inline .github/CODEOWNERS]: If you are adding red and costal you should add cxe-prg as well.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -146,7 +146,7 @@ packages/react-icons-mdl2 @microsoft/cxe-red @microsoft/cxe-coastal
 packages/react-monaco-editor @microsoft/fluentui-v8-website
 packages/react-components/react-positioning @microsoft/teams-prg
 packages/react-components/react-overflow @microsoft/teams-prg
-packages/react-components/react-shared-contexts @microsoft/teams-prg
+packages/react-components/react-shared-contexts @microsoft/teams-prg @microsoft/cxe-red @microsoft/cxe-coastal
```
Comment (micahgodbolt) [inline .github/CODEOWNERS:149]:
```suggestion
packages/react-components/react-shared-contexts @microsoft/teams-prg @microsoft/cxe-red
```

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/77bb9ec1eeedef91315dee3750bd06b15168c693'>77bb9ec1eeedef91315dee3750bd06b15168c693</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by JustSlone** [comment]
@ling1726 @layershifter (for @miroslavstastny) are you OK adding more codeowners to this package so we can approve PRs like #27295 independent of timezones?

**Pull request merged**